build.html

```diff
@@ -15,46 +15,45 @@ <h2 class="aligncenter">Authentication</h2>
         <p class="email-error text-danger">\{{ securityError }}</p>
       </template>
       <button class="btn btn-primary verify-identity" v-bind:class="{ disabled: disableButton }" type="submit">\{{ sendValidationLabel }}</button>
-      <button v-if="storedEmail" v-on:click="showVerify" class="btn btn-link have-code">I have an authentication code</button>
+      <button v-if="storedEmail" v-on:click="showVerify" class="btn btn-link have-code">I have a verification code.</button>
```

Comment:
This is a  text link that goes underneath a button. I would forego the period `.`